nautobot_golden_config/filters.py

```diff
@@ -403,10 +398,10 @@ class Meta:
         """Boilerplate filter Meta data for Remediation Setting."""
 
         model = models.RemediationSetting
-        fields = ["id", "platform", "remediation_type"]
+        fields = ["id"]
```

Comment:
Making this change causes the following error when filtering by a remediation type that would return 0 results:
![Screenshot 2023-09-13 at 1 57 45 PM](https://github.com/nautobot/nautobot-plugin-golden-config/assets/10467633/14d61e8e-3604-4161-8f9a-a5b5f709eb89)

`hierconfig` is indeed a valid choice for filtering, but should return 0 results in this case - not an error.

Reply:
don't think this actually does anything different. The same thing happens regardless of what fields are listed. And oddly enough in both of these I'm seeing what you're suggesting as in my source code.

Reply:
disregard for a moment, read this as a suggestion but its from the actual PR changes.  Will test further.

Reply:
So after testing, this is the error result regardless of what is in `fields=`

Reply:
got a PR coming in for this.